### DISCUSSION TITLE
[A/L] (2021/22) Relazioni - Esercizio 8

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#280] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152319061-385508cb-23cb-4641-9874-ff89742aac88.png)

**Comment by LuginiAndrea** [comment]
$R$ non è una relazione di equivalenza in quanto non è mai riflessiva
Infatti, $\forall z \in \mathbf{Z}$: $z -z = 0$, quindi $zRz$ non sussiste mai
